README.md

```diff
@@ -81,6 +81,29 @@ To re-index Elasticsearch based on the current database contents, run:
 bin/rails search:reindex
 ```
 
+## Troubleshooting
+
+### Flush Redis
+```
+$ redis-cli flushall
+OK
+```
+
+Check the database size is 0:
+```
+$ redis-cli
+127.0.0.1:6379> dbsize
+(integer) 0
+```
+
+### Running the PackageSyncWorker sidekiq job attempts to sync non existent data
+
+Ensure in config/enviroments/development.rb `config.ckan_v26_base_url` is not `https://data.gov.uk` but pointing to your localhost e.g. `http://localhost:4000`
+
+See instructions on how to [flush redis](#flush-redis). You will also need to purge purge SOLR via CKAN, and clear the Publish database.
+
+ Then re-run sidekiq jobs individually.
```

Comment:
Could provide a link to the sidekiq jobs

Reply:
Thanks @kentsanggds , I've restructured the whole readme so that there are headings and I can now link to other sections from here.